rectools/models/sasrec.py

```diff
@@ -717,9 +732,9 @@ def on_train_start(self) -> None:
 
     def training_step(self, batch: torch.Tensor, batch_idx: int) -> torch.Tensor:
         """TODO"""
-        x, y, w = batch
-        logits = self.forward(x)  # [batch_size, session_max_len, n_items + 1]
         if self.loss == "softmax":
+            x, y, w = batch.values()
```

Comment:
Let's access values by keys. It will simplify debugging for user if something goes wrong in custom data preparator.